log.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/btcsuite/seelog"
 
 	"github.com/decred/dcrrpcclient"
+	"github.com/decred/dcrticketbuyer/ticketbuyer"
```

Comment:
ticketbuyer should be a dcrwallet package, not part of the dcrticketbuyer repo.  When dcrticketbuyer is updated to use the wallet's built in ticket purchasing, it can simply be an rpc client that interacts with the wallet's ticket purchaser over RPC.